tools/notification-configuration/identity-resolver/identity-resolver.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <RootNamespace>identity_resolver</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="2.2.0" />
+    <PackageReference Include="System.CommandLine.DragonFruit" Version="0.3.0-alpha.19317.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\github-codeowner-subscriber\github-codeowner-subscriber.csproj" />
```

Comment:
Should this reference the common project?

Reply:
I'm uncertain why this builds properly. I had thought this was what I did.

Reply:
Ah. working because `github-codeowner` -> `common`. This is fixed now. Thanks for the catch.